projects/core/src/user/store/reducers/order-return-request.reducer.ts

```diff
@@ -1,20 +1,17 @@
 import { ReturnRequest, ReturnRequestList } from '../../../model/order.model';
 import { UserActions } from '../actions/index';
-import { OrderReturnRequestState } from '../user-state';
 
-export const returnRequestInitialState: OrderReturnRequestState = {};
+export const returnRequestInitialState = {};
 
 export function returnRequestReducer(
   state = returnRequestInitialState,
   action: UserActions.OrderReturnRequestAction
-): OrderReturnRequestState {
+): ReturnRequest {
   switch (action.type) {
+    case UserActions.LOAD_ORDER_RETURN_REQUEST_SUCCESS:
```

Comment:
I'm probably missing something, but I don't think handling `LOAD_ORDER_RETURN_REQUEST_SUCCESS` in a custom reducer is required.  It is a `LoaderSuccessAction` and the meta reducer will already put the action payload and sign it to the loader state value. 


In fact, the same goes for `CREATE_ORDER_RETURN_REQUEST_SUCCES`.

Reply:
yes, you are right. I will remove this reducer.